docs/_static/custom.js

```diff
@@ -31,6 +45,16 @@ document.addEventListener('DOMContentLoaded', () => {
     // insert ourselves after the element
     parent.insertBefore(table, element.nextSibling);
   });
+
+  settings.forEach(([settingName, settingFunction]) => {
+    let settingValue = JSON.parse(localStorage.getItem(settingName));
```

Comment:
You might want to wrap this in a `try`/`catch` block because any errors thrown will halt the entire `forEach`.